packages/telegraf/extra/start_telegraf.sh

```diff
@@ -24,6 +24,14 @@ mkdir -p "${TELEGRAF_CONTAINERS_DIR}"
 # Migrate old containers dir to new location in case the cluster was upgraded.
 /opt/mesosphere/active/telegraf/tools/migrate_containers_dir.sh "${LEGACY_CONTAINERS_DIR}" "${TELEGRAF_CONTAINERS_DIR}"
 
+# Add symlinks to any additional configuration files in the user-editable Telegraf config directory
+shopt -s nullglob
+for file in ${TELEGRAF_EXTRA_CONFIG_DIR}*; do
+	ln -sfn $file ${TELEGRAF_CONFIG_DIR}$(basename "$file")
```

Comment:
What happens if there's a collision with a user-created config file?

I wonder if it would be a good idea to add something to the filenames of these symlinks that suggests they come from builtin config:
```bash
  ln -sfn "$file" "${TELEGRAF_EXTRA_CONFIG_DIR}${TELEGRAF_BUILTIN_CONFIG_PREFIX}$(basename "$file")"
```
where `TELEGRAF_BUILTIN_CONFIG_PREFIX` is defined in `gen/dcos-config.yaml` to be `dcos.` or something.

This may help prevent collisions with user config, and we can filter by that prefix when cleaning up symlinks, to make sure we don't touch something we don't own.

```bash
find ${TELEGRAF_CONFIG_DIR} -type l -maxdepth 1 -name "${TELEGRAF_BUILTIN_CONFIG_PREFIX}*" ! -exec test -e {} \; -exec rm {} \;
```

Even with this change, collisions are possible, so we should know what that looks like and how a user could detect and work around it.